docs/deployment.md

```diff
@@ -79,6 +79,7 @@ Options:
                                   5]
   --ssl-keyfile TEXT              SSL key file
   --ssl-certfile TEXT             SSL certificate file
+  --ssl-password TEXT             SSL key file password
```

Comment:
Thoughts on naming this `--ssl-keyfile-password`? Both for extra explicitness that this is related to `--ssl-keyfile`, as well as visually hinting that "this is a more advanced option, since its name is longer" :-)

```suggestion
  --ssl-keyfile-password TEXT             SSL key file password
```